kuma/core/admin.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import unicode_literals
```

Comment:
I like including this, it should help shake out any string issues before we switch.